Dockerfile

```diff
@@ -1,67 +1,25 @@
-# Build Chainlink with SGX
+# Build Chainlink
 FROM smartcontract/builder:1.0.1 as builder
 
-# Have to reintroduce ENV vars from Baidu's SGX SDK image
-ENV PATH /root/.cargo/bin:/go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/sgxsdk/bin:/opt/sgxsdk/bin/x64
-ENV LD_LIBRARY_PATH /opt/sgxsdk/sdk_libs
-ENV SGX_SDK /opt/sgxsdk
+# Have to reintroduce ENV vars from builder image
+ENV PATH /go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 
 ARG COMMIT_SHA
 ARG ENVIRONMENT
-ARG SGX_ENABLED
-ARG SGX_SIMULATION
 
 ADD . /go/src/github.com/smartcontractkit/chainlink
 WORKDIR /go/src/github.com/smartcontractkit/chainlink
 RUN make build
 
-# Final layer: ubuntu with aesm and chainlink binaries (executable + enclave)
+# Final layer: ubuntu with chainlink binary
```

Comment:
Could we use a smaller image here? Wondering if something like alpine could work or if we're tied to this layer because of the builder image?

Reply:
Nah we could return to the alpine base now that I've gone down the separate Dockerfile approach.